documentation/api/wire_format/report_format_v6.markdown

```diff
@@ -152,14 +151,12 @@ All keys are required.
 
 `"timestamp"` is the date/time at which the resource's events occurred.
 
-`"status"` is a string representing the outcome of the event; this should be one
-of `success`, `failure`, or `noop`.
+`"status"` is a string representing the outcome of the event; possible values are `success`, `failure`, and `noop`.
```

Comment:
I'll stop commenting on these semicolons. They seem like periods to me but we should be consistent one way or another.